src/lbaf/Execution/lbsInformAndTransferAlgorithm.py

```diff
@@ -170,7 +167,7 @@ def __execute_information_stage(self):
             # Report on known peers when requested
             for rank in rank_set:
                 self._logger.debug(
-                    f"Peers known to rank {rank.get_id()}: {[r_k.get_id() for r_k in k_p]}")
+                    f"Peers known to rank {r.get_id()}: {[r_k.get_id() for r_k in k_p]}")
```

Comment:
What motivated this change? It doesn't look right to me given that the loop is over `rank`

Reply:
Incorrect cherry-picking on my part. Pushing a fix.